pylib/cqlshlib/cql3handling.py

```diff
@@ -738,11 +738,12 @@ def working_on_keyspace(ctxt):
                     ;
 <whereClause> ::= <relation> ( "AND" <relation> )*
                 ;
-<relation> ::= [rel_lhs]=<cident> ( "[" <term> "]" )? ( "=" | "<" | ">" | "<=" | ">=" | "CONTAINS" ( "KEY" )? ) <term>
+<relation> ::= [rel_lhs]=<cident> ( "[" <term> "]" )? ( "=" | "<" | ">" | "<=" | ">=" | "CONTAINS" ( "KEY" )? | "NOT CONTAINS" ( "KEY" )? ) <term>
              | token="TOKEN" "(" [rel_tokname]=<cident>
                                  ( "," [rel_tokname]=<cident> )*
-                             ")" ("=" | "<" | ">" | "<=" | ">=") <tokenDefinition>
+                             ")" ("=" | "!=" | "<" | ">" | "<=" | ">=") <tokenDefinition>
              | [rel_lhs]=<cident> "IN" "(" <term> ( "," <term> )* ")"
+             | [rel_lhs]=<cident> "NOT IN" "(" <term> ( "," <term> )* ")"
```

Comment:
Actually the way IN was added, it was claiming that token function supports IN when the goal was to added to general where clause on top.
I moved the IN/NOT in on top and then we get the same result as for CONTAINS - no mention of NOT and KEY option, just the operator. If we want to start adding in auto complete all options, I think this should be separate ticket. Let's not overcomplicate here. WDYT?

Reply:
I don't think it exactly claims that it works, just that it is accepted by syntax, albeit it will produce a meaningful validation error. I'm not sure we want to change that, and for sure this ticket wouldn't be the place to do so.

Changing to `( "NOT" )? "IN"`, same as we do with `( "NOT" )? "CONTAINS"`, should make sure that the suggestion works the same as other suggestions, avoiding suggesting both `NOT` and `NOT IN` in the same set of suggestions.

Reply:
`I don't think it exactly claims that it works, just that it is accepted by syntax, albeit it will produce a meaningful validation error. I'm not sure we want to change that, and for sure this ticket wouldn't be the place to do so.`
The way it is it makes suggestion that IN would work after TOKEN, which is not the case. We do not support such syntax. I moved it up together with CONTAINS, the same way CONTAINS is added and now it is suggested with where clauses but not after TOKEN. I am testing now and I will push a branch soon so you can see the exact changes.

Reply:
I don't see that it currently suggests `IN` or `NOT IN` with token:
```
cqlsh> SELECT * FROM k.t WHERE TOKEN(k)
<   <=  =   >   >=
```
It's in a different line than `CONTAINS` but not the same line as `TOKEN`, maybe I'm missing something.

Reply:
Let me find how the hell I broke COLORING and I will push my branch

Reply:
I ended up doing it your way and now it is OK, I was the one at fault.